mindmeld/resource_loader.py

```diff
@@ -417,6 +417,38 @@ def _traverse_labeled_queries_files(
                     if re.match(file_pattern, os.path.basename(file_path)):
                         yield a_domain, an_intent, file_path
 
+    def get_all_file_paths(self):
+        """ Get a list of text file paths across all intents.
+
+        Returns:
+            list: A list of all file paths.
+        """
+        file_iter = self._traverse_labeled_queries_files(file_pattern=".*.txt")
+        return [filename for _, _, filename in file_iter]
+
+    def filter_file_paths(self, file_pattern=".*", file_paths=None):
```

Comment:
`get_all_file_paths` returns all text files, so why are we matching for all patterns here?

Reply:
Good point, I removed the default value for file_pattern